CONTRIBUTING.md

```diff
@@ -0,0 +1,96 @@
+# Contributing to pyscf-ipu
+
+This project is still evolving but at the moment is focused around a high-performance and easily hackable implementation of Gaussian basis set DFT.
+We hope this is useful for the generation of large-scale datasets needed
+for training machine-learning models. We are interested in hearing any and
+all feedback so feel free to raise any questions, bugs encountered, or enhancement requests as [Issues](https://github.com/graphcore-research/pyscf-ipu/issues).
+
+## Setting up a development environment
+We recommend using the conda package manager as this can automatically enable
+the Graphcore Poplar SDK.  This is particularly useful in VS Code which can automatically
+activate the conda environment in a variety of scenarios:
+* visual debugging
+* running quick experiments in an interactive Jupyter window
+* using VS code for Jupyter notebook development.
+
+The following assumes that you have already setup an install of conda and that
+the conda command is available on your system path.  Refer to your preferred conda
+installer:
+* [miniforge installation](https://github.com/conda-forge/miniforge#install)
+* [conda installation documentation](https://docs.conda.io/projects/conda/en/latest/user-guide/install/index.html).
+
+1. Create a new conda environment with the same python version as you OS.
+   For example, on ubuntu 20 use `python=3.8.10`
+   ```bash
+   conda create -n pyscf-ipu python=3.8.10
+   ```
+
+2. Confirm that you have the Poplar SDK installed on your machine and store the location
+   in an environment variable.  The following will test that the SDK is found and
+   configured correctly:
+   ```bash
+   export POPLAR_SDK=/path/to/sdk
+   source $POPLAR_SDK/enable
+   gc-monitor
+   ```
```

Comment:
```suggestion
2. Confirm that you have the Poplar SDK installed on your machine and store the location
   in a temporary shell variable.  The following will test that the SDK is found and
   configured correctly:
   ```bash
   TMP_POPLAR_SDK=/path/to/sdk
   source $TMP_POPLAR_SDK/enable
   gc-monitor
   ```

```